charts/drax/values.yaml

```diff
@@ -673,14 +692,56 @@ loki-stack:
       port: 3100
       targetPort: http-metrics
     config:
+      auth_enabled: true
       table_manager:
-        # Retention polocy for logs in Loki
+        # Retention policy for logs in Loki
         retention_deletes_enabled: true
         # Must be a multiple of 168h
         retention_period: 672h
       limits_config:
+        # Avoid querying of data beyond the retention period
+        max_query_lookback: 672h
+        # Applied globally for all tenants unless overridden
+        retention_period: 672h
         max_entries_limit_per_query: 10000
-
+        per_tenant_override_config: /home/loki/tenant_override_config.yaml
+    extraContainers:
+      - name: reverse-proxy
+        image: k8spin/loki-multi-tenant-proxy:v1.0.0
```

Comment:
We really don't want to use this.  It's not maintained and the last commit is already 4 years old by now... I'm also not sure if I understand the need for a reverse proxy currently...